react/ui-components/src/hoc/app-config-screen-builder/RenderConditionalField.js

```
@@ -0,0 +1,259 @@
+            type={"button"}
+            icon="AddIcon"
+            size={"small"}
+            variation={"teritiary"}
+            label={t("ADD_OPTIONS")}
+            disabled={disabled}
+            onClick={() =>
+              setDrawerState((prev) => ({
+                ...prev,
+                [cField?.bindTo]: prev?.[cField?.bindTo]
+                  ? [
+                      ...prev?.[cField?.bindTo],
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)